vendor/engines/bulk_email/app/inputs/bulk_email/subject_input.rb

```diff
@@ -0,0 +1,14 @@
+module BulkEmail
+
+  class SubjectInput < SimpleForm::Inputs::TextInput
+
+    def input(_wrapper_options)
+      template.content_tag(:div, class: "input-prepend") do
+        template.content_tag(:span, options[:prefix], class: "add-on") +
```

Comment:
Is there a way we can append these or `safe_join` so that we don't need the `html_safe` that's triggering a rubocop warning?